client/.env

```
@@ -1,2 +1,3 @@
 NODE_ENV=development
-NEXT_PUBLIC_USE_KATANA=true
\ No newline at end of file
+NEXT_PUBLIC_USE_KATANA=true
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_USE_KATANA key should go before the NODE_ENV key

(UnorderedKey)